package.json

```
@@ -2,7 +2,6 @@
     "name": "omt-odt-language",
     "version": "2.1.2",
```

The version number should be increased, this is used for the overall plugin version.
```suggestion
    "version": "2.2.0",
```